_benchmark/reference/workloads/test-procedures.md

```diff
@@ -0,0 +1,193 @@
+
+Operations defined in the `operations` element can be reused more than once in the schedule:
+
+```yml
+{
+  "operations": [
+    {
+      "name": "force-merge",
+      "operation-type": "force-merge"
+    },
+    {
+      "name": "match-all-query",
+      "operation-type": "search",
+      "body": {
+        "query": {
+          "match_all": {}
+        }
+      }
+    }
+  ],
+  "schedule": [
+    {
+      "operation": "force-merge",
+      "clients": 1
+    },
+    {
+      "operation": "match-all-query",
+      "clients": 4,
+      "warmup-iterations": 1000,
+      "iterations": 1000,
+      "target-throughput": 100
+    }
+  ]
+}
+```
+
+#### Defining operations inline
+
+If you don't want reuse an operation in the schedule, you can also define operations inside the `schedule` element, as shown in the following example:
+
+```yml
+{
+  "schedule": [
+    {
+      "operation": {
+        "name": "force-merge",
+        "operation-type": "force-merge"
+      },
+      "clients": 1
+    },
+    {
+      "operation": {
+        "name": "match-all-query",
+        "operation-type": "search",
+        "body": {
+          "query": {
+            "match_all": {}
+          }
+        }
+      },
+      "clients": 4,
+      "warmup-iterations": 1000,
+      "iterations": 1000,
+      "target-throughput": 100
+    }
+  ]
+}
+```
+
+### Task options
+
+Each task contains the following options.
+
+Parameter | Required | Type | Description
+:--- | :--- | :--- | :---
+`operation` | Yes | List | Refers to either the name of an operation, defined in the `operations` element, or includes the entire operation inline.
+`name` | No | String | Specifies a unique name for the task when multiple tasks use the same operation.
+`tags` | No | String | Unique identifiers that can be used to filter between tasks.clients (optional, defaults to 1): The number of clients that should execute a task concurrently.
+`clients` | No | Integer | The number of clients that concurrently run the task. Default is `1`.
+
+### Target options
+
+OpenSearch Benchmark requires one of the following options when running a task.
+
+`target-throughput` | No | Integer | Defines the benchmark mode. When not defined, OpenSearch Benchmark assumes that this is a throughput benchmark and runs the task as fast as possible. This useful batch operations, where it is more important to achieve the best throughput as opposed to better latency. When defined, the target specifies the number of requests per second over all clients. For example, if you specify `target-throughput: 1000` with 8 clients, it means that each client will issue 125 (= 1000 / 8) requests per second. 
+`target-interval` | No | Interval | Defines an internal of less 1 / target-throughput (in seconds) less than one operation per second. Define either target-throughput or target-interval but not both (otherwise Rally will raise an error).
```

Comment:
```suggestion
`target-interval` | No | Interval | Defines an internval of less 1 / target-throughput (in seconds) less than one operation per second. Define either target-throughput or target-interval but not both (otherwise Rally will raise an error).
```